arctic/chunkstore/chunkstore.py

```diff
@@ -110,6 +123,9 @@ def read(self, symbol, chunk_range=None):
         chunk_range: object
             corresponding range object for the specified chunker (for
             DateChunker it is a DateRange object)
+        no_filter: boolean
+            perform chunk level filtering on the data (see filter() in _chunker)
```

Comment:
note that i decided i didnt like calling it no_filter, so its no longer a negative and the default behavior is now true rather than false.